typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -155,12 +160,15 @@ export const hyperlaneContextAgentChainConfig: AgentChainConfig<
     polygonzkevm: true,
     redstone: true,
     // Out of caution around pointer contracts (https://www.docs.sei.io/dev-interoperability/pointer-contracts) not being compatible
-    // and the scraper not gracefully handling txs that may not exist via the eth RPC, we don't run the scraper.
+    // and the scraper
```

Comment:
nit: I assume this was a linter / prettier thing? if not, i guess revert? but if so might be nicer to have this line be as long as possible and have the short one be the final line

Reply:
Not sure how this happened, will revert